src/CardList/index.js

```diff
@@ -96,14 +116,20 @@ export default class ImageList extends Component {
   renderGrid(items) {
     let { layout, columnCount, editor, _fonts } = this.props
 
-    let { fullWidth } = this.state
-    let width = fullWidth / columnCount - 8
+    const rows = this.getRows(items)
+    const columnWidth = { width: `${100 / columnCount}%` }
```

Comment:
nitpick: would it be worth calling this something like `cellContainerStyles`, or something similar, since the object could potentially contain any number of styles?
Alternatively, setting it directly on the `<View>` would also be fine.